### PR TITLE
add global style for material toggle btn

### PR DESCRIPTION
**Pull request by SlakvichAleksandr**

### CODE REVIEW

File: frontend/src/styles/material.sass [hunk 1]
```diff
@@ -43,10 +43,28 @@ snack-bar-container
...
+    height: 16px !important
+
+.mat-slide-toggle-thumb-container
+    top: -2px !important
+
+.mat-slide-toggle.mat-checked:not(.mat-disabled) .mat-slide-toggle-bar
+    background-color: $royal-blue
+
+.mat-slide-toggle.mat-checked:not(.mat-disabled) .mat-slide-toggle-thumb
+    background-color: $white
```
Comment (tatianahutii) [inline frontend/src/styles/material.sass]: Why do you define classes 2 times?

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/src/styles/material.sass (hunk 1 above)
Comment (SlakvichAleksandr) [reply frontend/src/styles/material.sass]: I have been trying for 2 hours but didn't find how this style can work in another implementation (

Comment (SlakvichAleksandr) [reply frontend/src/styles/material.sass]: Fix toggle disabled color (figma). I didn't find any way to not define classes 2 times for get this style

**Review by tatianahutii — APPROVED** [review]
LGTM

**Pull request merged**